platform/src/main/java/org/stellar/anchor/platform/configurator/DataConfigAdapter.java

```diff
@@ -0,0 +1,182 @@
+ * spring.datasource.url:                          JDBC URL of the database.
+ * spring.datasource.username:                     Login username of the database.
+ * spring.datasource.password:                     Login password of the database.
+ * spring.datasource.generate-unique-name:         Whether to generate a random datasource name.
+ * </pre>
+ *
+ * Spring Connection Pool Management
+ *
+ * <pre>
+ * spring.datasource.hikari.connection-timeout = 20000 # maximum number of milliseconds that a client will wait for a connection
+ * spring.datasource.hikari.minimum-idle= 10           # minimum number of idle connections maintained by HikariCP in a connection pool
+ * spring.datasource.hikari.maximum-pool-size= 10      # maximum pool size
+ * spring.datasource.hikari.idle-timeout=10000         # maximum idle time for connection
+ * spring.datasource.hikari.max-lifetime= 1000         # maximum lifetime in milliseconds of a connection in the pool after it is closed.
+ * spring.datasource.hikari.auto-commit =true          # default auto-commit behavior.
+ * </pre>
+ *
+ * Spring JPA.
+ *
+ * <pre>
+ * spring.jpa.database: 	                           Target database to operate on, auto-detected by default. Can be alternatively set using the "databasePlatform" property.
+ * spring.jpa.database-platform:                       Name of the target database to operate on, auto-detected by default. Can be alternatively set using the "Database" enum.
+ * spring.jpa.defer-datasource-initialization:         Whether to defer DataSource initialization until after any EntityManagerFactory beans have been created and initialized.: false
+ * spring.jpa.generate-ddl:                            Whether to initialize the schema on startup. : false
+ * spring.jpa.hibernate.ddl-auto:                      DDL mode. This is actually a shortcut for the "hibernate.hbm2ddl.auto" property. Defaults to "create-drop" when using an embedded database and no schema manager was detected. Otherwise, defaults to "none".
+ * spring.jpa.hibernate.naming.implicit-strategy:      Fully qualified name of the implicit naming strategy.
+ * spring.jpa.hibernate.naming.physical-strategy:      Fully qualified name of the physical naming strategy.
+ * spring.jpa.hibernate.use-new-id-generator-mappings: Whether to use Hibernate's newer IdentifierGenerator for AUTO, TABLE and SEQUENCE. This is actually a shortcut for the "hibernate.id.new_generator_mappings" property. When not specified will default to "true".
+ * spring.jpa.mapping-resources:                       Mapping resources (equivalent to "mapping-file" entries in persistence.xml).
+ * spring.jpa.open-in-view:                            Register OpenEntityManagerInViewInterceptor. Binds a JPA EntityManager to the thread for the entire processing of the request. : true
+ * spring.jpa.show-sql:                                Show SQL : false
+ * </pre>
+ */
+public class DataConfigAdapter extends SpringConfigAdapter {
```

Comment:
It may seem we should use `Database` because we only use database at this moment. But this adapter actually addresses how data is handled in the platform. Database is one of the implementations of how the data is handled.